File: app/src/main/java/com/yp/payment/model/ShopConfig.java
package com.yp.payment.model;

public class ShopConfig {
    private Integer shopId;
    private Integer cashierDeskId;
    private String shopName;
    private String username;
    private Integer autoLogin;
    private Integer autoPrint;

    public Integer getAutoPrint() {
        return autoPrint;
    }

    public void setAutoPrint(Integer autoPrint) {
        this.autoPrint = autoPrint;
    }

    public Integer getAutoLogin() {
        return autoLogin;
    }

    public void setAutoLogin(Integer autoLogin) {
        this.autoLogin = autoLogin;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getShopName() {
        return shopName;
    }

    public void setShopName(String shopName) {
        this.shopName = shopName;
    }

    public Integer getShopId() {
        return shopId;
    }

    public void setShopId(Integer shopId) {
        this.shopId = shopId;
    }

    public Integer getCashierDeskId() {
        return cashierDeskId;
    }

    public void setCashierDeskId(Integer cashierDeskId) {
        this.cashierDeskId = cashierDeskId;
    }
}

File: objectbox/src/main/java/com/usr/objectbox/objectbox/ShopConfig.kt
package com.usr.objectbox.objectbox

import io.objectbox.annotation.Entity
import io.objectbox.annotation.Id

@Entity
class ShopConfig {
    @Id
    var id: Long = 0
    var shopId: Int = 0
    var cashierDeskId: Int = 0
    var shopName: String = ""
    var username: String = ""
    var autoLogin: Int = 1
    var autoPrint: Int = 0

    override fun toString(): String {
        return "ShopConfig(id=$id, shopId=$shopId, cashierDeskId=$cashierDeskId, shopName='$shopName', username='$username', autoLogin=$autoLogin, autoPrint=$autoPrint)"
    }


}